### PR TITLE
14828 add cod population tables to pipeline

### PR DESCRIPTION
**Pull request by lgudyma-kontur**
fixing name for result datasets

### CODE REVIEW

File: scripts/build_ckan_dataset_description.py [hunk 1]
```diff
@@ -152,6 +149,9 @@ def build_ckan_dataset_description(strOutputFileName, number_of_objects, strFilt
     elif source == "gmted2010":
         ckan_dataset_description["url"] = "https://www.usgs.gov/"
         ckan_dataset_description["license_id"] = "other-pd"
+    elif source in ("ocha3w", "unocha", "unfis"):
```
Comment (lgudyma-kontur) [inline scripts/build_ckan_dataset_description.py:152]: @Zkir take a look if it is ok?

**Comment by lgudyma-kontur** [comment]
For pak and som there were no date in file description so I did not add year.

File: scripts/build_ckan_dataset_description.py (hunk 1 above)
Comment (Zkir) [reply scripts/build_ckan_dataset_description.py:152]: seems to be OK.

**Review by Zkir — APPROVED** [review]
Seems to be OK

**Pull request merged**